### PR TITLE
Issue #62 Fixing Che for v3.7.0

### LINKED ISSUE #62: Che addon is failing with OpenShift 3.7.0
### With OpenShift 3.6.0

```
$ minishift status
Minishift:  Running
Profile:    os_360
OpenShift:  Running (openshift v3.6.0+c4dd4cf)
DiskUsage:  11% of 17.9G

$ minishift addons apply che
-- Applying addon 'che':
[CHE] Create Image Stream.
[CHE] Create the Che server Template.
[CHE] Creating mini-che project.
[CHE] Switching to mini-che....
[CHE] Deploying Che on minishift.
Please wait while the pods all startup!
You can watch in the OpenShift console via:
  minishift console
Then you should be able the open the Che dashboard here:
  http://che-mini-che.192.168.42.21.nip.io
```

### With OpenShift 3.7.0
```
$ minishift status
Minishift:  Running
Profile:    os_370
OpenShift:  Running (openshift v3.7.0+7ed6862)
DiskUsage:  8% of 17.9G

$ minishift addon apply che
-- Applying addon 'che':
[CHE] Create Image Stream.
[CHE] Create the Che server Template.
[CHE] Creating mini-che project.Error from server (Forbidden): You may not request a new project via this API.
Error applying the add-on: Error executing command oc new-project mini-che --description="Eclipse Che on minishift" --as=developer.
```

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Fixes #62

### CODE REVIEW

File: add-ons/che/che.addon.remove [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Name: che
 # Description: Remove add on
 # Url: https://www.eclipse.org/che/docs/setup/openshift/index.html
+# Required-Vars: NAMESPACE
+# Var-Defaults: NAMESPACE=mini-che
 
 echo [CHE] Removing Image Stream
 oc delete -f streams/che-server-streams.json -n openshift
```
Comment (budhrg) [inline add-ons/che/che.addon.remove]: @LalatenduMohanty  Required below changes at line 13-14.

```
echo [CHE] Removing #{NAMESPACE} project
oc delete project #{NAMESPACE}
```

Comment (LalatenduMohanty) [reply add-ons/che/che.addon.remove]: @budhrg It is already there.

Comment (LalatenduMohanty) [reply add-ons/che/che.addon.remove]: @budhrg Replaced teh hard coded value of `mini-che` by #{NAMESPACE}

Comment (budhrg) [reply add-ons/che/che.addon.remove]: :+1:

**Pull request merged**